src/views/signup/components/survey/SurveyOneComponent.vue

```diff
@@ -94,37 +94,34 @@
 import { reactive, ref } from 'vue';
 
 import AlertModal from '@/components/AlertModal.vue';
-
-import { INITIAL_QUESTION_LIST } from '../../constants/question';
+import { SURVEY_LIST } from '@/views/signup/constants/survey';
 
 const emit = defineEmits(['next']);
 
+const surveyList = SURVEY_LIST.filter(question => question.id <= 3);
+
 // 답변
-const answers = reactive({
-  age: null,
-  income: null,
-  save: null,
-  habit: null,
-});
+const answers = reactive(
+  surveyList.map(question => ({
+    [question.id]: null,
+  }))
+);
```

Comment:
```suggestion
const answers = reactive({});
surveyList.forEach(q => {
  answers[q.id] = null;
});
```

원래의 코드를 실행하면
const answers = [
  { 1: null },
  { 2: 2 },
  { 3: 3 },
];
이렇게 저장되는데

const answers = {
  1: null,
  2: 2,
  3: 3,
};
혹시 이렇게 key-value 형태로 저장하려고 하신거라면 이렇게 해보면 어떨까 싶습니당

Reply:
하나의 배열로 초기 값들을 저장하려고 했는데 map함수의 내부 확인을 각각이 객체로 선언되는 것을 못 봤던 것 같습니다!!
제안해주신 방법을 참고해서 surveyList의 길이만큼의 배열로 초기화하도록 수정했습니다! 감사합니다👍
[9bae514](https://github.com/kb-jjukkumi/choogoomoneyna_FE/pull/44/commits/9bae514ffd17960c078330c5a35dcb56b6d98dd3)